conda_smithy/configure_feedstock.py

```diff
@@ -1842,11 +1843,22 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
             )
         )
 
-    for platform_arch in config["build_platform"].keys():
+    build_platforms = sorted(config["build_platform"].keys())
```

Comment:
The value of the dictionary is the build_platform and the key is the target_platform.
```suggestion
    target_platforms = sorted(config["build_platform"].keys())
```

Reply:
bc22a0036